css/excercise_one/part-b/excercise_one.css

```
@@ -0,0 +1,318 @@
+* {
+  color: black;
+  font-family: 'Poppins',  sans-serif;
+  text-decoration: none;
+}
+
+.s-pt {
+  padding-top: 8px;
+}
+
+.s-pb {
+  padding-bottom: 8px;
+}
+
+.m-pr {
+  padding-right: 40px;
+}
+
+.m-pb {
+  padding-bottom: 40px;
+}
+
+.m-pl {
+  padding-left: 40px;
+}
+
+.l-pt {
+  padding-top: 70px;
+}
+
+.l-pr {
+  padding-right: 70px;
+}
+
+.l-pb {
+  padding-bottom: 70px;
+}
+
+.l-pl {
+  padding-left: 70px;
+}
+
+.xl-pt {
+  padding-top: 120px;
+}
+
+.xl-pr {
+  padding-right: 120px;
+}
+
+.xl-pb {
+  padding-bottom: 120px;
+}
```

for some of them, sush as this, if you are using them both together all the time, you can create a specific class that contain both, on bootstrap for example they have  a padding or margin x and y that are for bottom and top, or right and left.

it would be something like:
```suggestion
.px-xl {
  padding-right: 120px;
  padding-bottom: 120px;
}
```

---

you can still use it the way you did it